istioctl/cmd/wait.go

```diff
@@ -125,6 +125,7 @@ func waitCmd() *cobra.Command {
 		},
 		Args: func(cmd *cobra.Command, args []string) error {
 			if err := cobra.ExactArgs(2)(cmd, args); err != nil {
+				cmd.Println(cmd.UsageString())
```

Comment:
I don't really understand why you would only revise `internal-debug` and `wait`. This is not consistent with other istioctl commands.

Reply:
I read all the cmd commands and found that if the parameters are complex, if there is an error in the input parameters, the Usage will be output to prompt the user. I think this is user-friendly, but only internal-debug and wait does not provide Usage, so I think it can be added here

Reply:
For commands like:
```
istioctl kube-inject
Error: 1 error occurred:
	* filename not specified (see --filename or -f)

istioctl validate
Error: error: you must specify resources by --filename.
Example resource specifications include:
   '-f rsrc.yaml'
   '--filename=rsrc.json'
```
They all return some kind of errors without the usage, and currently there's no command that prints the usage and then the error together.

It's not quite clear for users to see the error message if the usage text is also printed IMO.